src/test/java/it/gov/pagopa/rtd/ms/enrolledpaymentinstrument/infrastructure/kafka/ack/KafkaEnrollNotifyServiceTest.java

```diff
@@ -56,7 +56,7 @@ class KafkaEnrollAckServiceTest {
   private StreamBridge bridge;
   @Autowired
   private CorrelationIdService correlationIdService;
-  private KafkaEnrollAckService kafkaEnrollAckService;
```

Comment:
Not influent on code working flow, but maybe also variable name must change from ack to notify, mustn't it?🤔